src/parsers/manifestjson.js

```diff
@@ -791,6 +792,35 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  validateDeprecatedBrowserStyle() {
+    if (this.parsedJSON.manifest_version !== 3) {
```

Comment:
nit: this is a bit unusual. Shouldn't it be `< 3`?

Reply:
Then it would continue to warn in MV4 etc. As the comment on the next line emphasizes, this check is meant to cover MV2->MV3 migration.